dipy/viz/ui.py

```diff
@@ -3029,7 +3028,7 @@ def _setup(self):
         self.panel.add_element(self.up_button, pos, anchor="center")
 
         arrow_down = read_viz_icons(fname="arrow-down.png")
-        self.down_button = Button2D([("down", arrow_down)])
+        self.down_button = Button2D([("down", arrow_down)]
```

Comment:
small typo here, you forgot to close `)`. I am surprised that pep8speak did not complain.